src/reason_pprint_ast.ml

```diff
@@ -630,9 +630,6 @@ let rules = [
     (TokenPrecedence, (fun s -> (Right, s = "asr")));
   ];
   [
-    (TokenPrecedence, (fun s -> (Left, s.[0] == '#')));
```

Comment:
Oh, it's misleading because it uses `below_DOT_AND_SHARP` in the actual parse rule. So I think my suggestion for where its location in the list should be still stands, right?